parameters.yaml

```diff
@@ -35,9 +35,18 @@ custom_metadata_data_type:
   in: query
   name: data_type
   description: Data Type of Custom Metadata Property
-  required: false
+  required: true
```

Comment:
by doing this, it is also mandatory in the `index` call, right? I'm pretty sure we don't want that.

Reply:
this is actually confusing since it's rendered also in create/update CM pages and it's shown as not required which is not true

Reply:
yeah, I'm afraid it can't be reused as a single reference then.

Reply:
wait, perhaps it can be specified using `required` array

Reply:
which one?

Reply:
nah, forget it. you can't reuse the same reference.